### PR TITLE
feat(date-range): adds support for date range query

### PR DESCRIPTION
**Pull request by sak3tp**

### CODE REVIEW

File: lib/index.js [hunk 1]
```diff
@@ -247,6 +247,20 @@ var parseQuery = function (query, options) {
           var dateEnd = new Date(dateStart.getTime() + 86400000);
           var dateRange = {'$gte': dateStart, '$lt': dateEnd};
           addCondition(lcKey, operator == 'dne' ? { '$not': dateRange }: dateRange);
+        } else if (operator === 'dir') {
+          const dateRangeSeparator = ' to ';
```
Comment (Agrumas) [inline lib/index.js]: what `dir` means?  Maybe just `dr` - date range
Space isn't good separator especially in the URL due encodings. Simple coma would work. Range - `start,end`

Comment (sak3tp) [reply lib/index.js]: `dir` for date in range, sounds similar to `$in`.

About space, makes sense. Will make the change.

Comment (sak3tp) [reply lib/index.js]: @Agrumas , PTAL

**Review by Agrumas — APPROVED** [review]

**Comment by sak3tp** [comment]
@Agrumas Can you please merge it as well?

**Pull request merged**